Gems/PhysX/Code/Source/Scene/PhysXScene.cpp

```diff
@@ -1225,10 +1225,10 @@ namespace PhysX
         AZ_PROFILE_DATAPOINT(Physics, stats.getNbBroadPhaseRemoves(), RootCategory, BroadphaseSubCategory, "BroadPhaseRemoves");
 
         // Compute pair stats for all geometry types
+#if AZ_PROFILE_DATAPOINT
         AZ::u32 ccdPairs = 0;
```

Comment:
Notifying @SergeyAMZN and @amzn-sean about the PhysXScene changes to see if they are correct.

Reply:
looks good! it allows to avoid calculating the pairs if their profiling is disabled anyway. 
Thanks @nemerle !